Update upgrade-elastic-agent.asciidoc (#2774)
elastic/observability-docs: docs/en/ingest-management/fleet/upgrade-elastic-agent.asciidoc

image::images/upgrade-single-agent.png[Menu for upgrading a single {agent}]
+
The **Upgrade agent** option is grayed out if an upgrade is unavailable or

> the {kib} version is not higher or equal than the agent version.

Reviewer: ```suggestion
the {kib} version is below the agent version.
```
I think `below` is equivalent but a easier to parse than `not higher or equal than`.

Author: IMHO `below`  is not enough. In your case the kibana version can be equal the agent version and the option is grayed out.

Suggestion:
`the {kib} version is below or equal to the agent version.`

Reviewer: Sounds good to me! Thanks.

> the {kib} version is not higher or equal than the agent version.

Reviewer: ```suggestion
the {kib} version is below or equal to the agent version.
```
